compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -693,6 +695,18 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         _REGISTERED_FXEXECUTORS.pop(id(self), None)
         log.debug("%r: shutdown finished (thread: %s)", self, thread_id)
 
+    def get_hardware_details(self) -> str:
```

Comment:
You might consider renaming to `get_worker_hardware_details` or something else more descriptive.